packages/web-app-files/src/helpers/resource/actions/transfer.ts

```diff
@@ -141,10 +143,19 @@ export class ResourceTransfer extends ConflictDialog {
   }
 
   // This is for an edge case if a user moves a subfolder with the same name as the parent folder into the parent of the parent folder (which is not possible because of the backend)
-  public isOverwritingParentFolder(resource, targetFolder, targetFolderResources) {
+  public isOverwritingParentFolder(
+    resource: Resource,
+    targetFolder: Resource,
+    targetFolderResources: Resource[]
+  ) {
     if (resource.type !== 'folder') {
       return false
     }
+
+    if (targetFolder.path === unref(this.currentFolder)?.path) {
```

Comment:
Would it make sense to check on the `id` prop instead? Or can that lead to false results?

Reply:
It would work, I guess, though I feel like checking for `path` is a little more robust in this context, since identical paths are what cause this issue.